astropy/utils/metadata/core.py

```diff
@@ -0,0 +1,206 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""Classes for handling metadata."""
+
+import inspect
+from collections import OrderedDict
+from collections.abc import Mapping
+from copy import deepcopy
+
+__all__ = []
```

Comment:
Why not follow precedent in `angles` and `representations` and define here what is public at the top level? For better or worse, that is the current standard...

Reply:
Done.